packages/android_alarm_manager_plus/pubspec.yaml

```diff
@@ -5,6 +5,9 @@ version: 4.0.3
 homepage: https://plus.fluttercommunity.dev/
 repository: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/android_alarm_manager_plus
 issue_tracker: https://github.com/fluttercommunity/plus_plugins/labels/android_alarm_manager_plus
+topics:
+  - android
+  - alarm
```

Comment:
All other packages have `utils` topic but not this (and device info plus). Should this be added?

Reply:
Yes, missed it. Thanks.